src/renderer/components/Weather/TodayWeather.vue

```diff
@@ -0,0 +1,66 @@
+<template>
+  <section>
+    <header>
+      <p>{{day.date}}</p>
+      <Clock />
+    </header>
+    <img style="width: 200px;" :src="day.icon" />
+    <h1>{{city}}</h1>
+    <ul class="temperatures">
+      <li>{{day.temperatureMin}}°C</li>
+      <li>{{day.temperatureMax}}°C</li>
```

Comment:
I would put emphasis on the highest temperature of the day. What do you think about this ?

Reply:
I like the idea !